lib/Utilities.ts

```diff
@@ -282,3 +282,40 @@ export function escapeDiscordMarkdown(text: string) {
         },
     );
 }
+
+export async function handleFloorPriceChannel(client: any)  {
+    try {
+        let myChannel = client.channels.cache.get(config.priceChannel);
+        let data = await handleGetFromME();
```

Comment:
again doesn't really matter but if you're not changing something default to `const`